static/app/components/core/alert.chonk.tsx

```diff
@@ -0,0 +1,126 @@
+import {css, type DO_NOT_USE_ChonkTheme} from '@emotion/react';
+
+import type {AlertProps} from 'sentry/components/alert';
+import type {ChonkPropMapping} from 'sentry/utils/theme/withChonk';
+
+const chonkAlertPropMapping: ChonkPropMapping<AlertProps, ChonkAlertProps> = props => {
+  return {
+    ...props,
+    type:
+      props.type === 'muted' ? 'subtle' : props.type === 'error' ? 'danger' : props.type,
+  };
+};
+
+interface ChonkAlertProps extends Omit<AlertProps, 'type'> {
+  theme: DO_NOT_USE_ChonkTheme;
+  type: 'subtle' | 'info' | 'warning' | 'success' | 'danger';
+  size?: 'sm';
+}
+
+function Container({type, theme, size, ...props}: ChonkAlertProps) {
+  const colors = makeChonkAlertTheme(type, theme);
+  const padding = makeChonkAlertPadding(size, theme);
+
+  return (
+    <div
+      {...props}
+      css={css`
+        background-color: ${colors.background};
+        color: ${colors.color};
+        border: ${colors.border};
+        border-width: ${props.system ? '0px 0px 2px 0px' : '2px'};
+        border-radius: ${props.system ? '0px' : theme.borderRadius};
+        padding: ${padding};
+
+        cursor: ${props.expand ? 'pointer' : 'default'};
+
+        display: grid;
+        grid-template-columns:
+          ${!!props.showIcon && `minmax(0, max-content)`}
+          minmax(0, 1fr)
+          ${!!props.trailingItems && `max-content`}
+          ${!!props.expand && `max-content`};
+
+        gap: ${theme.space.md};
+
+        a,
+        button {
+          color: inherit;
+        }
+
+        a:hover,
+        button:hover {
+          color: inherit;
+        }
+
+        a {
+          text-decoration: underline;
+        }
+      `}
+    >
+      {props.children}
+    </div>
+  );
+}
+
+function makeChonkAlertPadding(
+  size: ChonkAlertProps['size'],
+  theme: DO_NOT_USE_ChonkTheme
+) {
+  switch (size) {
+    case 'sm':
+      return `${theme.space.mini} ${theme.space.md}`;
+    default:
+      return `${theme.space.md} ${theme.space.lg}`;
+  }
+}
+
+function makeChonkAlertTheme(
+  type: ChonkAlertProps['type'],
+  theme: DO_NOT_USE_ChonkTheme
+) {
+  // @TODO(jonasbadalic): implement opaque variants? We dont seem to use them anywhere
+  switch (type) {
```

Comment:
we also talked about removing the `theme.alert` palette - I can see that it’s still there, but the mappings created here look conceptually similar to what we have in `theme.alert`. As far as I can now see, `theme.alert` is not used anymore within the new alert component?

Reply:
Yeah, I do want to remove it, but there are a ton of places that use it, so the change is not trivial